model/ExpandedLicensing/Classes/NoAssertionLicense.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# NoAssertionLicense
+
+## Summary
+
+A License when no assertion can be made about its actual value.
```

Comment:
To align with the NoneLicense class text below, should this say something like:

"A concrete subclass of IndividualLicense used by the NoAssertionLicense Individual in the ExpandedLicensing profile."

Reply:
I'll align with the text below.